src/pyenphase/updaters/production.py

```diff
@@ -82,6 +96,8 @@ async def probe(
                     if type_["type"] == "eim" and type_["activeCount"]:
                         self._supported_features |= SupportedFeatures.METERING
                         self._supported_features |= SupportedFeatures.PRODUCTION
+                        if lines := type_.get("lines"):
```

Comment:
Why does this have a type ignore?

Reply:
Mypy complained on it not being Sized. It all worked but was caught in commit. Tried various options, but with my limited fundamental understanding of it I ended with that for now. Lines is [{},{},{}] of {str, str | float | int}.

![afbeelding](https://github.com/pyenphase/pyenphase/assets/120491684/a54fca50-6528-4a0d-8177-64a775fc2560)

Reply:
Added Any type to definition, and assert for list with #  nosec when type checking. That passed and looks like nosec is used in more spots.

Reply:
If you add a type for `lines` you shouldn't need the ignore.

`Any` effectively turns off the type checking